bot_management/bot_body.py

```diff
@@ -0,0 +1,126 @@
+from time import sleep, ctime
+import helpfull_functions
+from binance.client import Client
```

Comment:
We were using `CCXT` as this is more general library to handle a range of different exchanges. But in this case to speed up your work we can use directly a `binance` packakge. But this is not recommended.